test/test_helper.rb

```diff
@@ -12,14 +12,18 @@
 
 require 'dynflow'
 require 'dynflow/testing'
-begin require 'pry'; rescue LoadError; nil end
-
 require 'support/code_workflow_example'
 require 'support/middleware_example'
 require 'support/rescue_example'
 require 'support/dummy_example'
 require 'support/test_execution_log'
 
+begin
+  require 'pry'
+rescue LoadError
+  puts "Pry not installed. Skipping."
+end
+
```

Comment:
Had to make this change due to https://github.com/bbatsov/rubocop/issues/2119 as it's not released yet.